.eslintrc

```diff
@@ -1,2 +1 @@
-{ "extends": "scality", "env": { "jest": true } }
-
+{ "extends": "scality", "env": { "jest": true, "browser": true } }
```

Comment:
apparently to ensure that it works in both node and the browser we would need the `shared-node-browser` env

Reply:
I was just looking it up..yup.